### ISSUE TITLE
Feature request: Alternative to archive

### ISSUE THREAD

**Issue by daveaglick** [spectresystems/ghostly-beta#4]
I like the idea of being able to archive notifications, but I wonder how well that will scale over time. One option is to allow me to straight up delete notifications and forget about them. That's probably what I'd do with most of mine given the option.

**Comment by patriksvensson** [comment]
I understand the need for this, but would probably need a way to "resurrect" a deleted issue? Or am I overthinking this?

**Comment by daveaglick** [comment]
Yeah, I see your point. If I accidentally "delete" an issue, I probably don't want to hear "too bad, it's gone for good". That suggests there might need to be another "Deleted" tab. It sounds like issues then fall into a few categories:

* Inbox - issues with new notifications
  * Unless the issue has been deleted
* Archived - issues that have been archived and are expected to be searched/filtered/etc. later, I've read them and don't need it _right now_ but keep it around
* Deleted - issues I really don't care about and don't want to ever see again

What happens when I archive an issue but then it gets a new notification? Does that move the issue back into my inbox (at first thought, I feel like it should)? If not, there might need to be a way to say "get this issue out of my inbox but bring it back if there's a new notification". Maybe that's another good distinction between archived and deleted: archived issues can move back to the inbox on new notifications, deleted ones can't.

**Comment by daveaglick** [comment]
> What happens when I archive an issue but then it gets a new notification?

Nevermind, just got a notification on an archived issue and saw that it makes the issue bold/unread but leaves it in the archive. I think that's fine behavior now that I see it. That means the inbox is only for _new_ issues that I haven't archived yet, correct?

**Comment by patriksvensson** [comment]
> Nevermind, just got a notification on an archived issue and saw that it makes the issue bold/unread but leaves it in the archive. I think that's fine behavior now that I see it. That means the inbox is only for _new_ issues that I haven't archived yet, correct?

Yes, this is correct.

Would it makes sense to have the following top categories?

* Inbox
* Archived
* Starred
* Deleted

My original plan was to have a "Mute" feature where you could mute items in any state (inbox, starred, archived) and where you would not receive any notifications at all for the item.

Perhaps items in the "Deleted" state could be "emptied" (like trash in an email program) and add additional functionality to manually re-import individual items when needed.
